docs/distributing-an-application/installing-with-swarm.md

```diff
@@ -8,30 +8,60 @@ aliases = [
 ]
 +++
 
-We distribute an installation script that can be used to install Replicated into a new Swarm cluster. The cluster does not have to be created at this point, the Replicated install script will install Docker Engine and provision a new swarm cluster.
+We distribute an installation script that can be used to install Replicated into a new or existing Swarm cluster. The cluster does not have to be created at this point, the Replicated install script can install the Docker Engine and provision a new swarm cluster.
 
+### Basic install (recommended):
 
-### Basic Install
+The basic install will install Docker (as needed) and Replicated. It will save the install script to a file which you can inspect and then run. We recommend reading and understanding the install script prior to running.
 
-The basic swarm install requires the host is running Docker with a version between  {{< swarm_docker_version_minimum >}} - {{< swarm_docker_version_default >}}.
 
-Save the Docker Compose YAML to a file and run. We recommend reading and understanding the Compose file prior to running.
+```shell
+curl -sSL -o install.sh  https://get.replicated.com/swarm-init
+sudo bash ./install.sh
+```
+
+### Quick Install:  
+
+The quick swarm install will install Docker (as needed) and Replicated. Use this method if you have no need to view/change the installer script and you just want a one-line install.
 
 ```shell
-curl -sSL -o docker-compose.yml https://get.replicated.com/docker-compose.yml
-docker stack deploy -c docker-compose.yml replicated
+curl -sSL https://get.replicated.com/swarm-init | sudo bash
 ```
 
-### Easy install:
+### Advanced Install:
 
-The easy install will install Docker (as needed) and Replicated.
+The advanced swarm install requires the host is running Docker with a version between  {{< swarm_docker_version_minimum >}} - {{< swarm_docker_version_default >}}.
 
+This method will save the Docker Compose YAML to a file and then run a command using the YAML file as the input. We recommend reading and understanding the Compose file prior to running.
 
 ```shell
-curl -sSL https://get.replicated.com/swarm-init | sudo bash
+curl -sSL -o docker-compose.yml https://get.replicated.com/docker-compose.yml
+docker stack deploy -c docker-compose.yml replicated
+```
+### Flags:
```

Comment:
I think these flags should be listed in the above section (the install script), not in the compose yaml?